frappe/translations/fr.csv

```diff
@@ -799,9 +799,9 @@ DESC,DESC,
 Daily Event Digest is sent for Calendar Events where reminders are set.,Un Récapitulatif Quotidien est envoyé pour les Événements du Calendrier ayant des rappels.,
 Danger,Danger,
 Dark Color,Couleur sombre,
-Dashboard Chart,Tableau de bord,
-Dashboard Chart Link,Lien de tableau de bord,
-Dashboard Chart Source,Source du graphique du tableau de bord,
+Dashboard Chart,Tableau de bord - indicateurs,
+Dashboard Chart Link,Lien de tableau de bord - indicateurs,
+Dashboard Chart Source,Source du graphique du tableau de bord - indicateurs,
```

Comment:
according to google `indicateurs` means `indicators`

![Screenshot 2022-05-24 at 5 44 43 PM](https://user-images.githubusercontent.com/32034600/170032351-21c31614-201f-40c1-8828-2122c5686d8f.png)

which kind of changes the meaning for things (?)

Reply:
Actually Dashboard and Dashboard Charts have the same french translation "Tableau de bord" (Dashboard), it's really confusing 

![image](https://user-images.githubusercontent.com/1050053/170116866-10ecb288-afbe-4332-ba28-69a54abbdb2d.png)


the additions of "- **indicateurs**" mean Dashboard "detail/cards", like in KPI (Key performance **Indicator**: **indicateurs** clef de performance), that what Dashboard Charts are, a little database query that show some relevant data to show what going one into ERPNext (and hopefully in company)

Reply:
I've just remove the last "s" to remove plural, ERPNext link show always singular terms, no need to label them plural